templates/gson/Targets.cshtml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    @foreach (var art in @Model.MavenArtifacts) {
+    <AndroidJavaLibrary Include="$(MSBuildThisFileDirectory)..\..\jar\@(art.MavenArtifactId)-@(art.MavenArtifactVersion).jar" />
```

Comment:
Check `$(AndroidApplication)`.

Reply:
I'm not sure I understand why we are adding these `Targets.cshtml` files in the first place.  My understanding is we do not need them because these packages are using `<EmbeddedJar>`.  We are not including the loose `.jar` file in the NuGet, so these `Include`s point to a file that doesn't exist.

Reply:
kotlin - `InputJar`
kotlinx - `InputJar` also
other template sets are `EmbeddedJar`.

Should I make them `InputJar` and maybe we could share artifact and make nuget smaller?

Reply:
otherwise it would be embedded twice... Or am I wrong?

Reply:
I think we should focus this PR on fixing #439, and if there are additional unrelated changes we want to make we can handle them in separate PRs.